dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/GeneralWorkflowIT.java

```diff
@@ -1075,4 +1075,51 @@ public void testGenerateDOIFrozenVersion() throws ApiException {
         }
 
     }
+
+    /**
+    * Tests publishing/unpublishing workflows with the --newEntryName parameter
+    */
+    @Test
+    public void testPublishWithNewEntryName() {
```

Comment:
Would be good to have tests that verify both --newEntryName and --entryName work (all these tests are for workflow).

Reply:
Adding tests for both `--entryname` and `--newEntryName` for tools and workflows.